__tests__/applications/applications.test.js

```diff
@@ -129,6 +146,12 @@ describe('Applications page', () => {
     expect(applicationCards.length).toBe(6);
   });
 
+  it('should render the index of pending applications under dev flag === true', async function () {
+    await page.goto(`${SITE_URL}/applications?dev=true&status=pending`);
+    const indexOfApplication = await page.$$('.user-index');
```

Comment:
instead of class can we do it with data test-id?

Reply:
everywhere else we are using class for testing that's why i used class, to keep consistent with rest of the tests.

Reply:
yes but we should use data test id for this

Reply:
made the changes, can you check them please?